benchmark/single-source/DataBenchmarks.swift

```diff
@@ -416,7 +416,7 @@ func withUnsafeMutableBytes(_ N: Int, data: Data) {
 @inline(never)
 func copyBytes(_ N: Int, data: Data) {
   let amount = data.count
-  var buffer = UnsafeMutablePointer<UInt8>.allocate(capacity: amount)
+  let buffer = UnsafeMutablePointer<UInt8>.allocate(capacity: amount)
```

Comment:
@palimondo: could you please review this small change? I'm not sure whether usage of `var` was intentional.

Reply:
I don’t recall specific reason… if it complies, `let` Is fine :-)